quiz.txt

```diff
@@ -0,0 +1,22 @@
+#4. Create salary.csv file which contain sid,sname,salary.
+  #Display the employee record whose name begins from "S? also show no. of employee with first letter "S? out of total record.
+
+import csv
+a=open('c:\\sqlite3\\salary.csv','w',newline='')
+b=csv.writer(a)
+c=['sid','sname','salary']
+b.writerow(c)
+d=[[1,'om',10000],[2,'sai',50000],[3,'ram',20000],[4,'umashiv',30000],[5,'radekrishna',20000]]
+b.writerows(d)

```

Comment:
what is difference between writerow and writerows

Reply:
writerow:
its use to write a single row like a c=[-----------]

writerows:
its use to write many rows like a d=[[-------],[--------],[-------]]